cli/test/lib/tasks/download_spec.js

```diff
@@ -335,110 +335,115 @@ describe('lib/tasks/download', function () {
     })
   })
 
-  it('errors on too many redirects', function () {
-    nock('https://aws.amazon.com')
-    .get('/some.zip')
-    .reply(200, () => {
-      return fs.createReadStream(examplePath)
-    })
+  it('errors on too many redirects', async function () {
```

Comment:
The changes to this test aren't directly due to the changes in the PR, but this test was actually creating an unawaited Promise previously that was interfering with async stuff added in `util.getRealArch`. I fixed the test by re-stubbing the redirects and awaiting the 2nd download.start.